graphql/database/missions/UserMissionModel.js

```diff
@@ -53,21 +55,33 @@ class UserMission extends BaseModel {
         .integer()
         .default(self.fieldDefaults.tabs)
         .description(`the number of tabs the user has contributed`),
-      longestTabStreak: types
-        .number()
-        .integer()
-        .default(self.fieldDefaults.longestTabStreak)
-        .description(`count of longest tab streak in current mission`),
-      currentTabStreak: types
-        .number()
-        .integer()
-        .default(self.fieldDefaults.currentTabStreak)
-        .description(`count of the current tab streak in current mission`),
+      tabStreak: types
+        .object({
+          longestTabStreak: types.number().integer(),
+          currentTabStreak: types.number().integer(),
```

Comment:
@jedtan To know if a tab is part of the current tab streak, would we check that `User.lastTabTimestamp` is <24 hours ago?

Reply:
Yep, or the recent day object on Max Tabs

Reply:
@jedtan Awesome, LGTM!

Reply:
(within the same object so you'll have the ground truth for streaks on one single model.